### PR TITLE
Use .gz for installation and 'collect' tarball files

### PR DESCRIPTION
**Pull request by anusudarsan**
Please review related PRs too:

https://github.td.teradata.com/center-for-hadoop/jenkins-scripts/pull/66
https://github.td.teradata.com/center-for-hadoop/hfab/pull/72

### CODE REVIEW

**Comment by cawallin** [comment]
Are we intending to add this to the release? If not, you should hold off on merging this till after we verify presto-admin on the appliance and re-release.

**Comment by anusudarsan** [comment]
@cawallin This is not going into the release. I will wait hold off on merging

**Comment by cawallin** [comment]
@anusudarsan -- I talked about this with Rebecca and it's ok to push it. we should just make sure that Jenkins runs against the bug fix patch

**Comment by anusudarsan** [comment]
ok ..waiting for the travis build!

File: docs/installation/presto-admin-installation.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ work.
 
 To install ``presto-admin``:
 
-1. Copy the installer ``prestoadmin-<version>-offline.tar.bz2`` to the
+1. Copy the installer ``prestoadmin-<version>-offline.tar.gz`` to the
 location where you want ``presto-admin`` to run.
 Note that ``presto-admin`` does not have to be on the same node(s)
```
Comment (akshatnair) [inline docs/installation/presto-admin-installation.rst]: Change this line in second point
tar xvf prestoadmin-<version>-offline.tar.bz2
and check that the command to extract works with gz files

**Pull request merged**